pipeline-scripts/buildlib.groovy

```diff
@@ -173,16 +161,27 @@ def setup_venv() {
             throw err
         }
     }
-
     commonlib.shell(script: "rm -rf art-tools;  git clone -b ${branch_name} https://github.com/${owner}/art-tools.git")
-    commonlib.shell(script: "cd art-tools && ./install.sh")
```

Comment:
I would like us to keep using install.sh, it's a good way to keep localdev, aos-cd-jobs init and gh workflow in sync

Reply:
Done